specifyweb/frontend/templates/specify.html

```diff
@@ -3,6 +3,11 @@
 {% get_current_language as LANGUAGE_CODE %}
 <html lang="{{ LANGUAGE_CODE }}">
   <head>
+    <link rel="apple-touch-icon" sizes="180x180" href="/static/img/apple-touch-icon.png">
+    <link rel="mask-icon" href="/static/img/safari-pinned-tab.svg" color="#F79244">
+    <link rel="shortcut icon" href="/static/img/favicon.ico">
+    <link rel="icon" href="/static/img/favicon-512x512.png" sizes="any">
+    <link rel="icon" href="/static/img/short_logo.svg" type="image/svg+xml">
```

Comment:
Could you please move this to the end of the `</head>` tag (just after `<script>`?

This is because the `<meta charset="utf-8" />` line must be in the first KB of the file (there is a reason for it, don't remember). But also, we want the `main.js` line to begin executing as soon as possible as that's what actually displays the page

Reply:
Done– https://github.com/specify/specify7/pull/2752/commits/b80c26af456cff28c2071e5774bc5bf3cbac3922